frontend/public/kubevirt/models/vm.ts

```diff
@@ -40,6 +40,32 @@ export const VirtualMachineInstancePresetModel: K8sKind = {
   id: 'virtualmachineinstancepreset',
 };
 
+export const VirtualMachineInstanceReplicaSet: K8sKind = {
+  label: 'Virtual Machine Instance Replica Set',
+  labelPlural: 'Virtual Machine Replica Set',
```

Comment:
should be Virtual Machine Instance Replica Sets

Reply:
done